### ISSUE TITLE
warning icons ⚠️

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#632]
requires: https://github.com/cylc/cylc-flow/issues/4148

In the Cylc7 GUI there is a warning icon in the bottom right-hand corner which lights up whenever
a "warning" or higher is written to the log (I think it's warning, it might be "error" would have to look).
The user can click on the icon to open the log / dismiss it.

In the Cylc7 GScan there is a warning icon next to each workflow which appears next to the suite whenever
a task enters the failed or submit-failed statues. The user can then click on the icon to dismiss it.

For the Cylc8 GUI we should have a latest warnings stream from the Scheduler - https://github.com/cylc/cylc-flow/issues/4148

Suggest the following:

* Warnings should be displayed in GScan to the left of the task-state icons.
  * There is a warning icon component already implemented `@/components/cylc/WarningIcon`.
  * On hover a tooltip should list recent warnings.
  * On click the icon should be dismissed (turn greyscale, already implemented).
* The dashboard page should list these warnings.
  * We can probably do something more advanced there later.

**Pull requests welcome!**

**Comment by dpmatthews** [comment]
Duplicate of #535?

**Comment by oliver-sanders** [comment]
This issue leans more toward the Cylc 7 GScan warning icon (one per-workflow) which is activated by task failures.

#535 leans more toward the Cylc 7 GCylc warning icon (bottom right corner) which is activated by log messages ar WARNING level or higher.

I think log level should be the better option, although may require a bit of fiddling with log messages to ensure that recoverable warnings are emitted at INFO level to avoid false positives.

Closing as duplicate of #535
